AcadeMeData/test_msgboard.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from AcadeMeData.models import MessageBoards
+
+
+@pytest.fixture
+def generate_msgboard(id=1):
+    msgboard = MessageBoards(id=id, courseName="Linear Algebra")
+    msgboard.save()
+    return msgboard
+
+
+@pytest.mark.django_db
+class TestMessageBoardModel:
+    def test_create_msgboard(self, generate_msgboard):
+        board = generate_msgboard
```

Comment:
Please delete this line and thing how you could receive the board 
you already called the fixture in method declaration

Reply:
again...